src/inc/quic_platform_winkernel.h

```diff
@@ -99,6 +99,11 @@ ZwQueryInformationThread (
 
 #define INIT_NO_SAL(X) // No-op since Windows supports SAL
 
+#define CxPlatSocketError() WSAGetLastError()
+#define CxPlatCloseSocket(s) closesocket(s)
+#define CxPlatQuicErrorFromSocketError(e) HRESULT_FROM_WIN32(e)
+#define CxPlatAddressLengthType int
+
```

Comment:
These must not be used because they aren't correct.

1. Let's remove these.
2. I think it's best to move all these definitions these to a different (more scoped) header file, only for the stuff that needs them. Are they only used in one `.c` file? If so, put them at the top of that instead.

Reply:
all are moved to datapath_raw_socket.c